wls-gui-wahllokalsystem/src/components/wahlvorbereitung/WahlumgebungUwbCard.vue

```diff
@@ -0,0 +1,190 @@
+                label="Anzahl der Tische mit Sichtblenden"
+                clearable
+              />
+            </div>
+            <div>
+              <v-number-input
+                v-model="urnenwahlVorbereitung.anzahlNebenraeume"
+                class="mr-4"
+                :rules="[REQUIRED, MIN_NUMBER(0), MAX_NUMBER(99)]"
+                data-test="numberInputAnzahlNebenraeume"
+                label="Anzahl der Nebenräume im Wahlraum"
+                min-width="20rem"
+                clearable
+              />
+            </div>
+            <div>
+              <v-number-input
+                v-model="urnenwahlVorbereitung.anzahlWahlkabinen"
+                class="mr-4"
+                :rules="[REQUIRED, MIN_NUMBER(0), MAX_NUMBER(99)]"
+                data-test="numberInputAnzahlWahlkabinen"
+                label="Anzahl der Wahlkabinen"
+                min-width="20rem"
+                clearable
+              />
+            </div>
+          </div>
+        </v-form>
+      </v-card-text>
+      <v-card-actions>
+        <base-button-save
+          active
+          :disabled="isSaveButtonDisabled"
+          @click="onSaveWahlumgebungUWBClicked"
+        />
+      </v-card-actions>
+    </v-card>
+
+    <v-card
+      v-show="isMinimumRequired"
+      class="border-lg border-error"
+    >
+      <v-card-title>Ungültige Eingaben</v-card-title>
+      <v-card-text>
+        <div class="d-flex align-center mb-2">
+          <v-icon
+            color="red"
+            class="mr-2"
+            icon="$invalid"
+          />
+          <div class="text-red">
+            Die Summe der Kabinen, Tische und Nebenräume muss mindestens 1
+            betragen.
+          </div>
+        </div>
+      </v-card-text>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import type { Urnenwahlvorbereitung } from "@/types/wahlvorbereitung/Urnenwahlvorbereitung.ts";
+
+import { storeToRefs } from "pinia";
+import { computed, ref } from "vue";
+import {
+  VCard,
+  VCardActions,
+  VCardText,
+  VCardTitle,
+  VCheckbox,
+  VContainer,
+  VForm,
+  VIcon,
+  VNumberInput,
+} from "vuetify/components";
+
+import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import { useUserStore } from "@/stores/userStore.ts";
+import { useWahlbezirkStore } from "@/stores/wahlbezirkStore.ts";
+import { useWahlenStore } from "@/stores/wahlenStore.ts";
+import { MAX_NUMBER, MIN_NUMBER, REQUIRED } from "@/util/rules.ts";
+
+const { currentUserWahlbezirksArt } = storeToRefs(useUserStore());
+const anzahlWahlurnenValidForm = ref<null | boolean>(null);
+const wahlurnenForm = ref<HTMLFormElement | null>(null);
+const isCheckboxAlleVersiegeltEnabled = ref(false);
+const abstimmungsschutzvorrichtungenValidForm = ref<null | boolean>(null);
+const abstimmungsschutzvorrichtungenForm = ref<HTMLFormElement>();
+const wahlbezirkStore = useWahlbezirkStore();
```

Comment:
```suggestion
const { sendUrnenwahlvorbereitung } = useWahlbezirkStore();
```

und in zeile 180 statt `wahlbezirkStore.sendUrnenwahlvorbereitung(urnenwahlVorbereitung.value);` dann nur `sendUrnenwahlvorbereitung(urnenwahlVorbereitung.value);`